specification/storage/data-plane/Microsoft.FileStorage/preview/2021-12-02/file.json

```diff
@@ -5871,13 +5887,10 @@
           }
         },
         "DirectoryPath": {
-          "type": "string",
-          "xml": {
-            "attribute": true
-          }
+          "$ref": "#/definitions/StringEncoded"
```

Comment:
As mentioned in offline email... the DirectoryPath inside `EnumerationResults` will never be an Element and the `Encoded` property will actually come as a separate XML attribute for this case. So I think this may need to be changed to something like this:

```json
"Encoded":{
  "type": "boolean",
  "xml": {
    "attribute": true
  }
},
"DirectoryPath": {
  "type": "string",
  "xml": {
    "attribute": true
  }
},
```

This does change the client logic a little around this element but, as you mentioned in the OneNote, I don't think we actually return this so it should be okay and not require client-side changes (probably).

Reply:
Fixed!